src/boot/chid.c

```diff
@@ -14,9 +14,10 @@
  * https://github.com/aarch64-laptops/edk2/blob/dtbloader-app/EmbeddedPkg/Application/ConfigTableLoader/CHID.c
  */
 
+#include "efi.h"
```

Comment:
why is this moved?

Reply:
Please add a comment explaining it then, as headers are sorted alphabetically usually